plugins/smart_proxy_shellhooks/control

```diff
@@ -0,0 +1,21 @@
+Source: ruby-smart-proxy-shellhooks
+Section: ruby
+Priority: optional
+Maintainer: Lukas Zapletal <lzap+pub@redhat.com>
+Build-Depends: debhelper (>= 11~),
+               gem2deb
+Standards-Version: 4.3.0
+Homepage: http://github.com/theforeman/smart_proxy_shellhooks
+Testsuite: autopkgtest-pkg-ruby
+XS-Ruby-Versions: all
+Rules-Requires-Root: no
+
+Package: ruby-smart-proxy-shellhooks
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ruby | ruby-interpreter,
+         foreman-proxy (>= 2.20.0),
```

Comment:
your tardis is broken, we don't have 2.20 yet.

Reply:
Hmmm, thanks. Rebased.